packages/clangml-transforms/clangml-transforms.0.25/opam

```diff
@@ -13,6 +13,7 @@ depends: [
   "dolog" {>= "4.0.0" & < "5.0.0"}
   "traverse" {>= "0.2.0"}
   "refl" {>= "0.1.0"}
+  "ppxlib" {< "0.26.0"}
 ]
```

Comment:
This fails due to metapp failing (a dependency of traverse). There should be no dependency on ppxlib here.
See https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/a8aa66bb463131a112f5b73f2430e04ec04d82f2/variant/opam-2.1,compilers,4.13,ppxlib.0.26.0,revdeps,clangml-transforms.0.26